lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/GroupCommClient.cs

```diff
@@ -65,14 +63,9 @@ public class GroupCommClient : IGroupCommClient
             foreach (string serializedGroupConfig in groupConfigs)
             {
                 IConfiguration groupConfig = configSerializer.FromString(serializedGroupConfig);
-
-                IInjector injector = TangFactory.GetTang().NewInjector(groupConfig);
-                injector.BindVolatileParameter(GenericType<TaskConfigurationOptions.Identifier>.Class, taskId);
-                injector.BindVolatileInstance(GenericType<IGroupCommNetworkObserver>.Class, groupCommNetworkObserver);
-                injector.BindVolatileInstance(GenericType<NetworkService<GroupCommunicationMessage>>.Class, networkService);
-
-                ICommunicationGroupClient commGroup = injector.GetInstance<ICommunicationGroupClient>();
-                _commGroups[commGroup.GroupName] = commGroup;
+                IInjector groupInjector = injector.ForkInjector(groupConfig);
```

Comment:
This whole setup should be unneccesary once [REEF-289](https://issues.apache.org/jira/browse/REEF-289) is done. Then, we can have all of this in a context configuration, and the forking of Injectors will be in Evaluator code. We might want to make note of that future change here, e.g. by putting a comment with that issue ID here.